robottelo/ui/contentviews.py

```diff
@@ -55,18 +55,13 @@ def create(self, name, label=None, description=None, is_composite=False):
         self.wait_for_ajax()
         self.click(common_locators['create'])
 
-    def delete(self, name, really=True):
+    def delete(self, name):
```

Comment:
This PR will break the current working logic of content view delete.  I vote to leave content view delete as it is and fix in future PRs if necessary.